test/kyberNetwork.js

```diff
@@ -2809,14 +2809,15 @@ contract('KyberNetwork', function(accounts) {
             let srcAmountTwei = 136;
             let maxDestAmount = (new BigNumber(10)).pow(18);
 
-            buyRate = await network.getExpectedRate(tokenSrc.address, tokenDest.address, srcAmountTwei.valueOf());
```

Comment:
The changes here clashes with the same src and dest PR... Was working on multiple features on the same branch last time, so I think these changes were here by accident. Could u leave this file untouched? =)